source/js/modules/Modal.js

```diff
@@ -1,203 +1,230 @@
 
-        this._resetModal();
-        this._initializeModal();
-    }
+    this._resetModal();
+    this._initializeModal();
+  }
 
-    /**
+  /** Loads a new `ModalViewer` object and updates the `ModalHandler`
      *
      * @param {ModalViewer} modalViewer
      */
-    loadNewViewer(modalViewer) {
-        if (modalViewer) this.view = modalViewer;
-        else throw new Error('[ModalHandler] >>> loadNewViewer() >>> Object reference resulted in a nullptr');
+  loadNewViewer(modalViewer) {
+    if (modalViewer) this.view = modalViewer;
+    else throw new Error('[ModalHandler] >>> loadNewViewer() >>> Object reference resulted in a nullptr');
 
-        this._resetModal();
-        this._initializeModal();
-    }
+    this._resetModal();
+    this._initializeModal();
+  }
 
-    _initializeModal() {
-        if (this.container && this.view) {
-            if (this.view.txtBoxes.length < 1) this.view._addTextWrapper('first-text-wrapper').innerHTML += this.container.HtmlText;
-            else this.view.txtBoxes[0].innerHTML += this.container.HtmlText;
-        } else throw new Error('[ModalHandler] >>> _initializeModal() >>> Object reference resulted in a nullptr');
-    }
+  _initializeModal() {
+    if (this.container && this.view) {
+      if (this.view.txtBoxes.length < 1) this.view._addTextWrapper('first-text-wrapper').innerHTML += this.container.HtmlText;
+      else this.view.txtBoxes[0].innerHTML += this.container.HtmlText;
+    } else throw new Error('[ModalHandler] >>> _initializeModal() >>> Object reference resulted in a nullptr');
+  }
 
-    _resetModal() {
-        if (this.container && this.view) {
-            if (this.view.txtBoxes.length < 1) this.view._addTextWrapper('first-text-wrapper').innerHTML = '';
-            else {
-                // eslint-disable-next-line no-restricted-syntax
-                for (const txtbox of this.view.txtBoxes) txtbox.innerHTML = '';
-            }
-        } else throw new Error('[ModalHandler] >>> _initializeModal() >>> Object reference resulted in a nullptr');
-    }
+  _resetModal() {
+    if (this.container && this.view) {
+      if (this.view.txtBoxes.length < 1) this.view._addTextWrapper('first-text-wrapper').innerHTML = '';
+      else {
+        // eslint-disable-next-line no-restricted-syntax
+        for (const txtbox of this.view.txtBoxes) txtbox.innerHTML = '';
+      }
+    } else throw new Error('[ModalHandler] >>> _initializeModal() >>> Object reference resulted in a nullptr');
+  }
 
-    show() {
-        if (this.sliding) $(this.view.container.parentElement).slideDown(400);
-        else $(this.view.container.parentElement).fadeIn(400);
-        if (this.view.background && this.view.closeBtn) {
-            $(this.view.background).removeClass('no-display');
-            $(this.view.closeBtn).removeClass('no-display');
-        }
-        if (!this.bckg_scrolling) $('body').addClass('no-overflow');
+  /**
+     * `Show the modal-box`
```

Comment:
perhaps I'm missing something, but why are the backticks here?

Reply:
In Visual Studio Code it highlights the content inside of the backticks in documentation.